packages/shylu/shylu_dd/bddc/src/shylu_DofManager.h

```diff
@@ -265,37 +267,39 @@ template <class LO, class GO>
     indices.erase(iter, indices.end());
   }
 
-  static void generateRootMap(RCP<const Map> & inputMap,
-                              int root,
-                              RCP<const Map> & rootMap)
+  static void generateRootMap(RCP<const Map> & inputMap, 
```

Comment:
Tpetra has a `computeGatherMap` function that does this.

Reply:
Yes. I recall having problems using computeGatherMap in the past and wrote something to do the same thing. I can revisit this in the near future to see if computeGatherMap now does what I want.